aten/src/ATen/native/cuda/MultiTensorApply.cuh

```diff
@@ -57,9 +57,9 @@ template<> struct TensorListScalarListMetadata<c10::complex<double>, 1> {
 };
 
 template<> struct TensorListScalarListMetadata<c10::complex<double>, 2> {
-  const void* addresses[1][60];
-  int64_t numel_for_tensor[60];
-  c10::complex<double> scalar_vals[60];
+  const void* addresses[1][66];
```

Comment:
why are these values hardcoded and not taken form `depth_to_*` lists above?

Reply:
Yeah I understand the values have to be smaller, I'd still like to see them a few lines up with the rest, and not hardcoded here, for readability, you can create some new list if needed.